mat64/list_test.go

```diff
@@ -383,7 +384,8 @@ func testTwoInput(c *check.C,
 		&basicMatrix{},
 		&basicVectorer{},
 		&basicSymmetric{},
-		&basicTriangular{},
```

Comment:
I don't understand the rationale for this; at 380 and 381 we are already creating both triangles with valid uplo states.